src/package-lock.json

```diff
@@ -10664,4 +10664,4 @@
       }
     }
   }
-}
+}
```

Comment:
**undo change before merging** @erinysong I think you accidentally removed the endline here